modules/ui_extra_networks_checkpoints.py

```diff
@@ -14,7 +14,10 @@ def refresh(self):
         shared.refresh_checkpoints()
 
     def create_item(self, name, index=None, enable_filter=True):
-        checkpoint: sd_models.CheckpointInfo = sd_models.checkpoint_aliases.get(name)
+        checkpoint: sd_models.CheckpointInfo = sd_models.checkpoint_aliases.get(name, None)
+        if checkpoint is None():

```

Comment:
`None` is not callable ^^
```suggestion
        if checkpoint is None:

```

Reply:
yeah I noticed that just now
must have accidentally press the key when just before I was committing

Reply:
fixed